src/transformers/models/bert/modeling_flax_bert.py

```diff
@@ -300,22 +300,28 @@ def __call__(self, input_ids, attention_mask, token_type_ids, position_ids):
         return encoder, pooled
 
 
+class FlaxBertPretrainedModel(FlaxPreTrainedModel):
```

Comment:
I think we should keep the design as similar as possible to PyTorch as we can. Also, in Flax I would prefer to use composition over inheritance -> so **not** really how it's done here: https://github.com/huggingface/transformers/blob/4b1f56b65919a41752820301d9bf17dac31bb52d/src/transformers/models/bert/modeling_flax_bert.py#L628 . I'd prefer to rather have `FlaxBertModel` as an attribute of `FlaxBertForMaskedLM` as it's done for PyTorch.